lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParameters.py

```diff
@@ -120,7 +123,8 @@ def test_no_shape_parameters_exception(self):
         plugin = Plugin(distribution="truncnorm")
         msg = "For the truncated normal distribution"
         with self.assertRaisesRegex(ValueError, msg):
-            plugin._rescale_shape_parameters(self.mean, self.std)
+            plugin._rescale_shape_parameters(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _rescale_shape_parameters of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)